plugins/generator-1.19.4/forge-1.19.4/templates/modbase/mod.java.ftl

```diff
@@ -44,6 +44,10 @@ import org.apache.logging.log4j.Logger;
 			${JavaModName}Fluids.REGISTRY.register(bus);
 			${JavaModName}FluidTypes.REGISTRY.register(bus);
 		</#if>
+    <#if w.hasElementsOfType("villagerprofession")>

```

Comment:
This is incorrectly indented, recheck indentations in your PR

Reply:
Done!